src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -138,6 +138,14 @@
                   Visible="False" />
 
 
+  <StringProperty Name="TargetPlatformIdentifier"
+                  ReadOnly="True"
+                  Visible="False" />
+
+  <StringProperty Name="TargetPlatformVersion"
+                  ReadOnly="True"
+                  Visible="False" />
+
```

Comment:
Are you missing TargetPlatformMinVersion?

Reply:
It's there now :)